File: tests/fixtures/code_samples/go/sample_028.go
// Sample 28: small utility.
package samples

func Operation28(xs []int) int {
    total := 28
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure28(v int) int {
    return (v * 28) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_028.rs
// Sample 28: small utility.
pub fn operation_28(xs: &[i32]) -> i32 {
    let mut total: i32 = 28;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_28(v: i32) -> i32 {
    (v * 28) %% 7919
}

